paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -47,7 +47,11 @@ import java.util.Locale
 
 @Suppress("unused")
 public class PaparazziPlugin : Plugin<Project> {
+  private lateinit var providerFactory: ProviderFactory
```

Comment:
As far as I know you can make the plugin abstract and use field injection for standard grade types

Reply:
Yea, was feeling lazy, but that's a good call out.  Updated to constructor injection.